### PR TITLE
AC_WPNav: Fix calculation of _scurve_jerk_time using radians.

### PR DESCRIPTION
**Pull request by lthall**
This is a fix of a simple mistake where I used the centi degree units rather than radians.

The bigger question is should I add cdss to the name of those functions?

### CODE REVIEW

**Comment by rmackay9** [comment]
@lthall, great thanks.  Presumably this should be back ported to 4.1 as well so I've added the 4.1 project.

We probably should add the _cdss to the name of the other methods as well but that could be done as a separate PR if you like.  Either way is fine.

File: libraries/AC_WPNav/AC_WPNav.cpp [hunk 1]
```diff
@@ -859,9 +859,9 @@ void AC_WPNav::calc_scurve_jerk_and_jerk_time()
     }
 
     // calculate jerk time
-    // jounce (the rate of change of jerk) uses the attitude control input time constant because multicopters
-    // lean to accelerate meaning the change in angle is equivalent to the change in acceleration
-    const float jounce = MIN(_attitude_control.get_accel_roll_max() * GRAVITY_MSS, _attitude_control.get_accel_pitch_max() * GRAVITY_MSS);
+    // Jounce (the rate of change of jerk) uses the attitude control input time constant because multicopters
+    // lean to accelerate. This means the change in angle is equivalent to the change in acceleration
+    const float jounce = MIN(_attitude_control.get_accel_roll_max_radss() * GRAVITY_MSS, _attitude_control.get_accel_pitch_max_radss() * GRAVITY_MSS);
```
Comment (IamPete1) [inline libraries/AC_WPNav/AC_WPNav.cpp:864]: I'm struggling to understand this. I feel like there should be some trig in here somewhere.

Comment (lthall) [reply libraries/AC_WPNav/AC_WPNav.cpp:864]: This is the small angle approximation for sin. Given that the minimum jerk happens with a constant roll rate at zero degrees this is perfect.

Comment (IamPete1) [reply libraries/AC_WPNav/AC_WPNav.cpp:864]: Thanks for the explanation.

**Comment by lthall** [comment]
> @lthall, great thanks. Presumably this should be back ported to 4.1 as well so I've added the 4.1 project.
> 
> We probably should add the _cdss to the name of the other methods as well but that could be done as a separate PR if you like. Either way is fine.

Yes, it should.

**Review by rmackay9 — APPROVED** [review]

**Comment by rmackay9** [comment]
Sorry in advance to those affected by merge conflicts but hopefully this will reduce future errors caused by confusion between centi-degrees and radians.

**Comment by rmackay9** [comment]
I could only include the first commit, "AC_WPNav: Fix calculation of _scurve_jerk_time using radians." in Copter-4.1 due to a string of merge conflicts

**Pull request merged**